### DISCUSSION TITLE
Home assistant Octopus energy integration only updates every 30 mins despite octopus mini (was working fine)...

### DISCUSSION THREAD

**Discussion by maniacgt** [BottlecapDave/HomeAssistant-OctopusEnergy#929] [General]
So I'm not sure when this happened, possibly a week or more ago but I've noticed my HA octopus entities only now updating every 30 mins where previously it's updated every minute thanks to the home mini option/key. I've reloaded the integration, restarted HA, refreshed the API key etc but still I'm only getting 30 minute updates (things like consumption and cost used to update every minute). Any other options/causes I could look into? Thanks a lot -- Gary

**Comment by joel-graham** [comment]
Not sure if I'm allowed to comment on there, but there's an ongoing thread here: https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/914

**Reply by maniacgt** [reply]
Fab, thank you. I did a quick search but clearly wasn't the right terms to bring this up.
